cluster/images/etcd/migrate/data_dir.go

```diff
@@ -144,12 +144,36 @@ func (v *VersionFile) Read() (*EtcdVersionPair, error) {
 	return vp, nil
 }
 
+// contains returns true iff VersionFile exists and contains given EtcdVersionPair.
+func (v *VersionFile) contains(vp *EtcdVersionPair) (bool, error) {
```

Comment:
I would actually call this function "equals" - effectively you check if its content equals to the given vp.

Reply:
I'd argue that I'm checking if this file *contains* given vp. I'd expect `equals` method to take another `VersionFile` and compare them somehow. With that said, I don't have strong opinion on that.

Reply:
I prefer equals, but I'm not going to fight until death for it.

Reply:
changed to `equals`